html/index.html

```diff
@@ -414,18 +414,18 @@ <h2>About <br> Myself</h2>
         
                 <div class="form-section">
         
-                    <form action="">
+                    <form action="https://formspree.io/f/mayzpyer" method="post">
         
-                        <input type="text" placeholder="Toyyib" class="text"> <br>
-                        <input type="email" placeholder="Simpleshaikh2@gmail.com" class="text"> <br>
-                        <textarea type="text" placeholder="write your message heres" id="type-msg"> </textarea><br>
+                        <input type="text" name="name" placeholder="Name" class="text" maxlength="30" required> <br>
+                        <input type="email" placeholder="Email" class="text" maxlength="30" required> <br>
+                        <textarea type="text" placeholder="write your message here" cols="30" rows="10" maxlength="500" id="type-msg" required> </textarea><br>
         
                     </form>
         
                     <div class="btn-sec">
         
                         <button class="sec-btn">   
-                            Get in touch
+                            <span>Get in touch</span>
                         </button>
```

Comment:
Good job on the design 💯 but some adjustments can be made. Two things I noticed mainly, the first one is that you have some gray background that is not part of the Figma, second thing is that the spacing between your inputs isn't quite the same as in the Figma, look to adjust these two things to improve the design. 🤓  Take a look at the differences 👇🏽 

## Your design

![Screenshot 2023-04-04 at 10 00 11](https://user-images.githubusercontent.com/45739963/229816908-2755efab-4637-4f39-8d9d-e767940bc1a7.png)

## Figma

![Screenshot 2023-04-04 at 10 00 24](https://user-images.githubusercontent.com/45739963/229816976-deb95646-5a2d-490d-bb12-9aa09da9711b.png)

Cheers 🥂 and happy coding 👨🏽‍💻